spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -46,7 +46,7 @@ def get_unit_waveforms(recording, sorting, unit_ids=None, channel_ids=None,
                 If True and 'grouping_property' is given, the property of each unit is assigned as the corresponding
                 property of the recording extractor channel on which the average waveform is the largest
             max_channels_per_waveforms: int or None
-                Maximum channels per waveforms to return. If None, all channels are returned
+                Maximum channels per waveforms to return. If None, all channels are returned.
             n_jobs: int
```

Comment:
@colehurwitz we need to coordinate on this ahhaah 

I usually go in and remove full stops ;)